Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py

```diff
@@ -1032,8 +1032,27 @@ def checkout(self, argument, prune=None):
                 cwd=self.root_path,
             ).returncode else self.commit()
 
+        match = self.dev_branches.match(argument)
         branch_remote = self.remote_for(argument)
-        if branch_remote:
+
+        # if branch has a remote and is not a dev branch
+        if not match and branch_remote:
+            path = '{}/{}'.format(branch_remote, argument)
+            sys.stderr.write("Resetting branch {} to remote {}.\n Checkout will erase all un-pushed changes on {}.\n".format(argument, path, argument))
+            if Terminal.choose(
+                'Ensure all desired changes have been saved. Continue with checkout?\n',
```

Comment:
I would have this say:
`"'{}' looks like a PR branch. Would you like to override the local version of this branch with the version from '{}'".format(argument, branch_remote)`

Reply:
As discussed, not relevant as PR branches are filtered out. (Also, significant changes in amended commit incoming.)